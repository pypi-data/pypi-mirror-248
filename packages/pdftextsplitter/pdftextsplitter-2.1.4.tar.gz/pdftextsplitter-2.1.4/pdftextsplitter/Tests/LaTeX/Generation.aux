\relax 
\providecommand\babel@aux[2]{}
\@nameuse{bbl@beforestart}
\bibstyle{ieeetr}
\babel@aux{english}{}
\@writefile{toc}{\contentsline {chapter}{\numberline {1}Some piece of text}{1}{}\protected@file@percent }
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\gdef \@abspage@last{1}
