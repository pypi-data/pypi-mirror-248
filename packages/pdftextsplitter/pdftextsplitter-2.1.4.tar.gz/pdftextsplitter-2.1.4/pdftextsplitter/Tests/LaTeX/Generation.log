This is pdfTeX, Version 3.141592653-2.6-1.40.22 (TeX Live 2022/dev/Debian) (preloaded format=pdflatex 2023.7.21)  13 NOV 2023 16:14
entering extended mode
 restricted \write18 enabled.
 %&-line parsing enabled.
**Generation.tex
(./Generation.tex
LaTeX2e <2021-11-15> patch level 1
L3 programming layer <2022-01-21>
(/usr/share/texlive/texmf-dist/tex/latex/base/report.cls
Document Class: report 2021/10/04 v1.4n Standard LaTeX document class
(/usr/share/texlive/texmf-dist/tex/latex/base/size10.clo
File: size10.clo 2021/10/04 v1.4n Standard LaTeX file (size option)
)
\c@part=\count185
\c@chapter=\count186
\c@section=\count187
\c@subsection=\count188
\c@subsubsection=\count189
\c@paragraph=\count190
\c@subparagraph=\count191
\c@figure=\count192
\c@table=\count193
\abovecaptionskip=\skip47
\belowcaptionskip=\skip48
\bibindent=\dimen138
)
(/usr/share/texlive/texmf-dist/tex/generic/babel/babel.sty
Package: babel 2022/01/26 3.70 The Babel package
\babel@savecnt=\count194
\U@D=\dimen139
\l@unhyphenated=\language87

(/usr/share/texlive/texmf-dist/tex/generic/babel/txtbabel.def)
\bbl@readstream=\read2
\bbl@dirlevel=\count195

(/usr/share/texlive/texmf-dist/tex/generic/babel-english/english.ldf
Language: english 2017/06/06 v3.3r English support from the babel system
Package babel Info: Hyphen rules for 'canadian' set to \l@english
(babel)             (\language0). Reported on input line 102.
Package babel Info: Hyphen rules for 'australian' set to \l@ukenglish
(babel)             (\language40). Reported on input line 105.
Package babel Info: Hyphen rules for 'newzealand' set to \l@ukenglish
(babel)             (\language40). Reported on input line 108.
))
(/usr/share/texlive/texmf-dist/tex/latex/amsmath/amsmath.sty
Package: amsmath 2021/10/15 v2.17l AMS math features
\@mathmargin=\skip49

For additional information on amsmath, use the `?' option.
(/usr/share/texlive/texmf-dist/tex/latex/amsmath/amstext.sty
Package: amstext 2021/08/26 v2.01 AMS text

(/usr/share/texlive/texmf-dist/tex/latex/amsmath/amsgen.sty
File: amsgen.sty 1999/11/30 v2.0 generic functions
\@emptytoks=\toks16
\ex@=\dimen140
))
(/usr/share/texlive/texmf-dist/tex/latex/amsmath/amsbsy.sty
Package: amsbsy 1999/11/29 v1.2d Bold Symbols
\pmbraise@=\dimen141
)
(/usr/share/texlive/texmf-dist/tex/latex/amsmath/amsopn.sty
Package: amsopn 2021/08/26 v2.02 operator names
)
\inf@bad=\count196
LaTeX Info: Redefining \frac on input line 234.
\uproot@=\count197
\leftroot@=\count198
LaTeX Info: Redefining \overline on input line 399.
\classnum@=\count199
\DOTSCASE@=\count266
LaTeX Info: Redefining \ldots on input line 496.
LaTeX Info: Redefining \dots on input line 499.
LaTeX Info: Redefining \cdots on input line 620.
\Mathstrutbox@=\box50
\strutbox@=\box51
\big@size=\dimen142
LaTeX Font Info:    Redeclaring font encoding OML on input line 743.
LaTeX Font Info:    Redeclaring font encoding OMS on input line 744.
\macc@depth=\count267
\c@MaxMatrixCols=\count268
\dotsspace@=\muskip16
\c@parentequation=\count269
\dspbrk@lvl=\count270
\tag@help=\toks17
\row@=\count271
\column@=\count272
\maxfields@=\count273
\andhelp@=\toks18
\eqnshift@=\dimen143
\alignsep@=\dimen144
\tagshift@=\dimen145
\tagwidth@=\dimen146
\totwidth@=\dimen147
\lineht@=\dimen148
\@envbody=\toks19
\multlinegap=\skip50
\multlinetaggap=\skip51
\mathdisplay@stack=\toks20
LaTeX Info: Redefining \[ on input line 2938.
LaTeX Info: Redefining \] on input line 2939.
)
(/usr/share/texlive/texmf-dist/tex/latex/amsfonts/amssymb.sty
Package: amssymb 2013/01/14 v3.01 AMS font symbols

(/usr/share/texlive/texmf-dist/tex/latex/amsfonts/amsfonts.sty
Package: amsfonts 2013/01/14 v3.01 Basic AMSFonts support
\symAMSa=\mathgroup4
\symAMSb=\mathgroup5
LaTeX Font Info:    Redeclaring math symbol \hbar on input line 98.
LaTeX Font Info:    Overwriting math alphabet `\mathfrak' in version `bold'
(Font)                  U/euf/m/n --> U/euf/b/n on input line 106.
))
(/usr/share/texlive/texmf-dist/tex/latex/graphics/epsfig.sty
Package: epsfig 2017/06/25 v1.7b (e)psfig emulation (SPQR)

(/usr/share/texlive/texmf-dist/tex/latex/graphics/graphicx.sty
Package: graphicx 2021/09/16 v1.2d Enhanced LaTeX Graphics (DPC,SPQR)

(/usr/share/texlive/texmf-dist/tex/latex/graphics/keyval.sty
Package: keyval 2014/10/28 v1.15 key=value parser (DPC)
\KV@toks@=\toks21
)
(/usr/share/texlive/texmf-dist/tex/latex/graphics/graphics.sty
Package: graphics 2021/03/04 v1.4d Standard LaTeX Graphics (DPC,SPQR)

(/usr/share/texlive/texmf-dist/tex/latex/graphics/trig.sty
Package: trig 2021/08/11 v1.11 sin cos tan (DPC)
)
(/usr/share/texlive/texmf-dist/tex/latex/graphics-cfg/graphics.cfg
File: graphics.cfg 2016/06/04 v1.11 sample graphics configuration
)
Package graphics Info: Driver file: pdftex.def on input line 107.

(/usr/share/texlive/texmf-dist/tex/latex/graphics-def/pdftex.def
File: pdftex.def 2020/10/05 v1.2a Graphics/color driver for pdftex
))
\Gin@req@height=\dimen149
\Gin@req@width=\dimen150
)
\epsfxsize=\dimen151
\epsfysize=\dimen152
)
(/usr/share/texlive/texmf-dist/tex/latex/parskip/parskip.sty
Package: parskip 2021-03-14 v2.0h non-zero parskip adjustments

(/usr/share/texlive/texmf-dist/tex/latex/kvoptions/kvoptions.sty
Package: kvoptions 2020-10-07 v3.14 Key value format for package options (HO)

(/usr/share/texlive/texmf-dist/tex/generic/ltxcmds/ltxcmds.sty
Package: ltxcmds 2020-05-10 v1.25 LaTeX kernel commands for general use (HO)
)
(/usr/share/texlive/texmf-dist/tex/generic/kvsetkeys/kvsetkeys.sty
Package: kvsetkeys 2019/12/15 v1.18 Key value parser (HO)
))
(/usr/share/texlive/texmf-dist/tex/latex/etoolbox/etoolbox.sty
Package: etoolbox 2020/10/05 v2.5k e-TeX tools for LaTeX (JAW)
\etb@tempcnta=\count274
))
(/usr/share/texlive/texmf-dist/tex/latex/booktabs/booktabs.sty
Package: booktabs 2020/01/12 v1.61803398 Publication quality tables
\heavyrulewidth=\dimen153
\lightrulewidth=\dimen154
\cmidrulewidth=\dimen155
\belowrulesep=\dimen156
\belowbottomsep=\dimen157
\aboverulesep=\dimen158
\abovetopsep=\dimen159
\cmidrulesep=\dimen160
\cmidrulekern=\dimen161
\defaultaddspace=\dimen162
\@cmidla=\count275
\@cmidlb=\count276
\@aboverulesep=\dimen163
\@belowrulesep=\dimen164
\@thisruleclass=\count277
\@lastruleclass=\count278
\@thisrulewidth=\dimen165
)
(/usr/share/texlive/texmf-dist/tex/latex/was/gensymb.sty
Package: gensymb 2003/07/02 v1.0 (WaS)
)
(/usr/share/texlive/texmf-dist/tex/latex/placeins/placeins.sty
Package: placeins 2005/04/18  v 2.2
)
(/usr/share/texlive/texmf-dist/tex/latex/caption/caption.sty
Package: caption 2020/10/26 v3.5g Customizing captions (AR)

(/usr/share/texlive/texmf-dist/tex/latex/caption/caption3.sty
Package: caption3 2020/10/21 v2.2e caption3 kernel (AR)
\captionmargin=\dimen166
\captionmargin@=\dimen167
\captionwidth=\dimen168
\caption@tempdima=\dimen169
\caption@indent=\dimen170
\caption@parindent=\dimen171
\caption@hangindent=\dimen172
Package caption Info: Standard document class detected.
)
\c@caption@flags=\count279
\c@continuedfloat=\count280
)
(/usr/share/texlive/texmf-dist/tex/latex/caption/subcaption.sty
Package: subcaption 2020/10/07 v1.3j Sub-captions (AR)
\c@subfigure=\count281
\c@subtable=\count282
)
(/usr/share/texlive/texmf-dist/tex/latex/base/inputenc.sty
Package: inputenc 2021/02/14 v1.3d Input encoding file
\inpenc@prehook=\toks22
\inpenc@posthook=\toks23
)
(/usr/share/texlive/texmf-dist/tex/latex/braket/braket.sty)
(/usr/share/texlive/texmf-dist/tex/latex/carlisle/slashed.sty
Package: slashed 1997/01/16 v0.01 Feynman Slashed Character Notation (DPC)
)
(/usr/share/texlive/texmf-dist/tex/latex/pdfpages/pdfpages.sty
Package: pdfpages 2022/01/29 v0.5u Insert pages of external PDF documents (AM)

(/usr/share/texlive/texmf-dist/tex/latex/base/ifthen.sty
Package: ifthen 2020/11/24 v1.1c Standard LaTeX ifthen package (DPC)
)
(/usr/share/texlive/texmf-dist/tex/latex/tools/calc.sty
Package: calc 2017/05/25 v4.3 Infix arithmetic (KKT,FJ)
\calc@Acount=\count283
\calc@Bcount=\count284
\calc@Adimen=\dimen173
\calc@Bdimen=\dimen174
\calc@Askip=\skip52
\calc@Bskip=\skip53
LaTeX Info: Redefining \setlength on input line 80.
LaTeX Info: Redefining \addtolength on input line 81.
\calc@Ccount=\count285
\calc@Cskip=\skip54
)
(/usr/share/texlive/texmf-dist/tex/latex/eso-pic/eso-pic.sty
Package: eso-pic 2020/10/14 v3.0a eso-pic (RN)
\ESO@tempdima=\dimen175
\ESO@tempdimb=\dimen176

(/usr/share/texlive/texmf-dist/tex/latex/xcolor/xcolor.sty
Package: xcolor 2021/10/31 v2.13 LaTeX color extensions (UK)

(/usr/share/texlive/texmf-dist/tex/latex/graphics-cfg/color.cfg
File: color.cfg 2016/01/02 v1.6 sample color configuration
)
Package xcolor Info: Driver file: pdftex.def on input line 227.
Package xcolor Info: Model `cmy' substituted by `cmy0' on input line 1352.
Package xcolor Info: Model `hsb' substituted by `rgb' on input line 1356.
Package xcolor Info: Model `RGB' extended on input line 1368.
Package xcolor Info: Model `HTML' substituted by `rgb' on input line 1370.
Package xcolor Info: Model `Hsb' substituted by `hsb' on input line 1371.
Package xcolor Info: Model `tHsb' substituted by `hsb' on input line 1372.
Package xcolor Info: Model `HSB' substituted by `hsb' on input line 1373.
Package xcolor Info: Model `Gray' substituted by `gray' on input line 1374.
Package xcolor Info: Model `wave' substituted by `hsb' on input line 1375.
))
\AM@pagewidth=\dimen177
\AM@pageheight=\dimen178
\AM@fboxrule=\dimen179

(/usr/share/texlive/texmf-dist/tex/latex/pdfpages/pppdftex.def
File: pppdftex.def 2022/01/29 v0.5u Pdfpages driver for pdfTeX (AM)
)
\pdfpages@includegraphics@status=\count286
\AM@pagebox=\box52
\AM@global@opts=\toks24
\AM@pagecnt=\count287
\AM@toc@title=\toks25
\c@AM@survey=\count288
\AM@templatesizebox=\box53
)
(/usr/share/texlive/texmf-dist/tex/latex/enumitem/enumitem.sty
Package: enumitem 2019/06/20 v3.9 Customized lists
\labelindent=\skip55
\enit@outerparindent=\dimen180
\enit@toks=\toks26
\enit@inbox=\box54
\enit@count@id=\count289
\enitdp@description=\count290
)
(/usr/share/texlive/texmf-dist/tex/latex/listings/listings.sty
\lst@mode=\count291
\lst@gtempboxa=\box55
\lst@token=\toks27
\lst@length=\count292
\lst@currlwidth=\dimen181
\lst@column=\count293
\lst@pos=\count294
\lst@lostspace=\dimen182
\lst@width=\dimen183
\lst@newlines=\count295
\lst@lineno=\count296
\lst@maxwidth=\dimen184

(/usr/share/texlive/texmf-dist/tex/latex/listings/lstmisc.sty
File: lstmisc.sty 2020/03/24 1.8d (Carsten Heinz)
\c@lstnumber=\count297
\lst@skipnumbers=\count298
\lst@framebox=\box56
)
(/usr/share/texlive/texmf-dist/tex/latex/listings/listings.cfg
File: listings.cfg 2020/03/24 1.8d listings configuration
))
Package: listings 2020/03/24 1.8d (Carsten Heinz)

(/usr/share/texlive/texmf-dist/tex/latex/listings/lstlang1.sty
File: lstlang1.sty 2020/03/24 1.8d listings language file
)
(/usr/share/texlive/texmf-dist/tex/latex/listings/lstlang1.sty
File: lstlang1.sty 2020/03/24 1.8d listings language file
)
(/usr/share/texlive/texmf-dist/tex/latex/listings/lstmisc.sty
File: lstmisc.sty 2020/03/24 1.8d (Carsten Heinz)
)
(/usr/share/texlive/texmf-dist/tex/latex/titlesec/titlesec.sty
Package: titlesec 2021/07/05 v2.14 Sectioning titles
\ttl@box=\box57
\beforetitleunit=\skip56
\aftertitleunit=\skip57
\ttl@plus=\dimen185
\ttl@minus=\dimen186
\ttl@toksa=\toks28
\titlewidth=\dimen187
\titlewidthlast=\dimen188
\titlewidthfirst=\dimen189
)
(/usr/share/texlive/texmf-dist/tex/latex/geometry/geometry.sty
Package: geometry 2020/01/02 v5.9 Page Geometry

(/usr/share/texlive/texmf-dist/tex/generic/iftex/ifvtex.sty
Package: ifvtex 2019/10/25 v1.7 ifvtex legacy package. Use iftex instead.

(/usr/share/texlive/texmf-dist/tex/generic/iftex/iftex.sty
Package: iftex 2020/03/06 v1.0d TeX engine tests
))
\Gm@cnth=\count299
\Gm@cntv=\count300
\c@Gm@tempcnt=\count301
\Gm@bindingoffset=\dimen190
\Gm@wd@mp=\dimen191
\Gm@odd@mp=\dimen192
\Gm@even@mp=\dimen193
\Gm@layoutwidth=\dimen194
\Gm@layoutheight=\dimen195
\Gm@layouthoffset=\dimen196
\Gm@layoutvoffset=\dimen197
\Gm@dimlist=\toks29
)
(/usr/share/texlive/texmf-dist/tex/latex/fancyhdr/fancyhdr.sty
Package: fancyhdr 2021/01/28 v4.0.1 Extensive control of page headers and foote
rs
\f@nch@headwidth=\skip58
\f@nch@O@elh=\skip59
\f@nch@O@erh=\skip60
\f@nch@O@olh=\skip61
\f@nch@O@orh=\skip62
\f@nch@O@elf=\skip63
\f@nch@O@erf=\skip64
\f@nch@O@olf=\skip65
\f@nch@O@orf=\skip66
)
(/usr/share/texlive/texmf-dist/tex/latex/natbib/natbib.sty
Package: natbib 2010/09/13 8.31b (PWD, AO)
\bibhang=\skip67
\bibsep=\skip68
LaTeX Info: Redefining \cite on input line 694.
\c@NAT@ctr=\count302
)
(/usr/share/texlive/texmf-dist/tex/latex/notoccite/notoccite.sty
Package: notoccite 2000/07/20
)
(/usr/share/texlive/texmf-dist/tex/latex/tocbibind/tocbibind.sty
Package: tocbibind 2010/10/13 v1.5k extra ToC listings
Package tocbibind Info: The document has chapter divisions on input line 50.


Package tocbibind Note: Using chapter style headings, unless overridden.

) (/usr/share/texlive/texmf-dist/tex/latex/l3backend/l3backend-pdftex.def
File: l3backend-pdftex.def 2022-01-12 L3 backend support: PDF output (pdfTeX)
\l__color_backend_stack_int=\count303
\l__pdf_internal_box=\box58
)
(./Generation.aux)
\openout1 = `Generation.aux'.

LaTeX Font Info:    Checking defaults for OML/cmm/m/it on input line 143.
LaTeX Font Info:    ... okay on input line 143.
LaTeX Font Info:    Checking defaults for OMS/cmsy/m/n on input line 143.
LaTeX Font Info:    ... okay on input line 143.
LaTeX Font Info:    Checking defaults for OT1/cmr/m/n on input line 143.
LaTeX Font Info:    ... okay on input line 143.
LaTeX Font Info:    Checking defaults for T1/cmr/m/n on input line 143.
LaTeX Font Info:    ... okay on input line 143.
LaTeX Font Info:    Checking defaults for TS1/cmr/m/n on input line 143.
LaTeX Font Info:    ... okay on input line 143.
LaTeX Font Info:    Checking defaults for OMX/cmex/m/n on input line 143.
LaTeX Font Info:    ... okay on input line 143.
LaTeX Font Info:    Checking defaults for U/cmr/m/n on input line 143.
LaTeX Font Info:    ... okay on input line 143.

(/usr/share/texlive/texmf-dist/tex/context/base/mkii/supp-pdf.mkii
[Loading MPS to PDF converter (version 2006.09.02).]
\scratchcounter=\count304
\scratchdimen=\dimen198
\scratchbox=\box59
\nofMPsegments=\count305
\nofMParguments=\count306
\everyMPshowfont=\toks30
\MPscratchCnt=\count307
\MPscratchDim=\dimen199
\MPnumerator=\count308
\makeMPintoPDFobject=\count309
\everyMPtoPDFconversion=\toks31
) (/usr/share/texlive/texmf-dist/tex/latex/epstopdf-pkg/epstopdf-base.sty
Package: epstopdf-base 2020-01-24 v2.11 Base part for package epstopdf

(/usr/share/texlive/texmf-dist/tex/generic/infwarerr/infwarerr.sty
Package: infwarerr 2019/12/03 v1.5 Providing info/warning/error messages (HO)
)
(/usr/share/texlive/texmf-dist/tex/latex/grfext/grfext.sty
Package: grfext 2019/12/03 v1.3 Manage graphics extensions (HO)

(/usr/share/texlive/texmf-dist/tex/generic/kvdefinekeys/kvdefinekeys.sty
Package: kvdefinekeys 2019-12-19 v1.6 Define keys (HO)
))
(/usr/share/texlive/texmf-dist/tex/generic/pdftexcmds/pdftexcmds.sty
Package: pdftexcmds 2020-06-27 v0.33 Utility functions of pdfTeX for LuaTeX (HO
)
Package pdftexcmds Info: \pdf@primitive is available.
Package pdftexcmds Info: \pdf@ifprimitive is available.
Package pdftexcmds Info: \pdfdraftmode found.
)
Package epstopdf-base Info: Redefining graphics rule for `.eps' on input line 4
85.
Package grfext Info: Graphics extension search list:
(grfext)             [.pdf,.png,.jpg,.mps,.jpeg,.jbig2,.jb2,.PDF,.PNG,.JPG,.JPE
G,.JBIG2,.JB2,.eps]
(grfext)             \AppendGraphicsExtensions on input line 504.

(/usr/share/texlive/texmf-dist/tex/latex/latexconfig/epstopdf-sys.cfg
File: epstopdf-sys.cfg 2010/07/13 v1.3 Configuration of (r)epstopdf for TeX Liv
e
))
LaTeX Info: Redefining \celsius on input line 143.
Package gensymb Info: Faking symbols for \degree and \celsius on input line 143
.


Package gensymb Warning: Not defining \perthousand.

LaTeX Info: Redefining \ohm on input line 143.
Package gensymb Info: Using \Omega for \ohm on input line 143.

Package gensymb Warning: Not defining \micro.

Package caption Info: Begin \AtBeginDocument code.
Package caption Info: listings package is loaded.
Package caption Info: End \AtBeginDocument code.
(/usr/share/texlive/texmf-dist/tex/latex/pdflscape/pdflscape.sty
Package: pdflscape 2019/12/05 v0.12 Display of landscape pages in PDF (HO)

(/usr/share/texlive/texmf-dist/tex/latex/graphics/lscape.sty
Package: lscape 2020/05/28 v3.02 Landscape Pages (DPC)
)
Package pdflscape Info: Auto-detected driver: pdftex on input line 81.
)
\c@lstlisting=\count310

*geometry* driver: auto-detecting
*geometry* detected driver: pdftex
*geometry* verbose mode - [ preamble ] result:
* driver: pdftex
* paper: custom
* layout: <same size as paper>
* layoutoffset:(h,v)=(0.0pt,0.0pt)
* modes: twoside 
* h-part:(L,W,R)=(56.9055pt, 369.88583pt, 56.9055pt)
* v-part:(T,H,B)=(79.66771pt, 546.29292pt, 56.9055pt)
* \paperwidth=483.69684pt
* \paperheight=682.86613pt
* \textwidth=369.88583pt
* \textheight=546.29292pt
* \oddsidemargin=-15.36449pt
* \evensidemargin=-15.36449pt
* \topmargin=-29.60228pt
* \headheight=12.0pt
* \headsep=14.22636pt
* \topskip=10.0pt
* \footskip=28.45274pt
* \marginparwidth=57.0pt
* \marginparsep=11.0pt
* \columnsep=10.0pt
* \skip\footins=9.0pt plus 4.0pt minus 2.0pt
* \hoffset=0.0pt
* \voffset=0.0pt
* \mag=1000
* \@twocolumnfalse
* \@twosidetrue
* \@mparswitchtrue
* \@reversemarginfalse
* (1in=72.27pt=25.4mm, 1cm=28.453pt)

Chapter 1.
[1


{/var/lib/texmf/fonts/map/pdftex/updmap/pdftex.map}] (./Generation.aux) ) 
Here is how much of TeX's memory you used:
 10530 strings out of 478287
 169637 string characters out of 5849289
 633608 words of memory out of 5000000
 28581 multiletter control sequences out of 15000+600000
 469865 words of font info for 30 fonts, out of 8000000 for 9000
 1141 hyphenation exceptions out of 8191
 70i,5n,76p,1199b,261s stack positions out of 5000i,500n,10000p,200000b,80000s
</usr/
share/texlive/texmf-dist/fonts/type1/public/amsfonts/cm/cmbx12.pfb></usr/share/
texlive/texmf-dist/fonts/type1/public/amsfonts/cm/cmr10.pfb>
Output written on Generation.pdf (1 page, 28501 bytes).
PDF statistics:
 18 PDF objects out of 1000 (max. 8388607)
 10 compressed objects within 1 object stream
 0 named destinations out of 1000 (max. 500000)
 1 words of extra memory for PDF output out of 10000 (max. 10000000)

