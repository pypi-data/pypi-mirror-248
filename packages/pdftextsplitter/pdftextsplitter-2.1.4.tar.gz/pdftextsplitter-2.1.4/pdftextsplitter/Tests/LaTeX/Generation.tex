\documentclass[10pt,a4paper]{report}

% use the packages I need:
\usepackage[english]{babel}             
\usepackage{amsmath}                   
\usepackage{amssymb}                     
\usepackage{epsfig} 
\usepackage{parskip}
\usepackage{booktabs}
\usepackage{gensymb}
\usepackage{graphicx} 
\usepackage{placeins}
\usepackage{caption}
\usepackage{subcaption}
\usepackage{inputenc}
\usepackage{braket} 
\usepackage{slashed} 
\usepackage{pdfpages}
\usepackage{enumitem} 

% Define code listings:
\usepackage{listings}
\lstset {
    language=C++,
    backgroundcolor=\color{black!5}, % set backgroundcolor
    basicstyle=\footnotesize,% basic font setting
}

% Define our own chapter titles:
\usepackage{titlesec}
\titleformat{\chapter}[hang]{\normalfont\raggedright\Huge\bfseries}{\thechapter ~~}{0pt}{\Huge}
\titlespacing*{\chapter}{0pt}{-40pt}{20pt}
\titlespacing*{\section}{0pt}{15pt}{0pt}
\titlespacing*{\subsection}{0pt}{5pt}{0pt}

% Define your own paper size:
\usepackage{geometry}
\geometry{papersize={170.0mm,240.0mm}} % absolute paper size.
\geometry{top=28.0mm}                  % Top marging.
\geometry{bottom=20.0mm}               % Bottom margin.
\geometry{left=20.0mm}                 % Left margin.
\geometry{right=20.0mm}                % Right margin.

% Headers & Layout:
\geometry{twoside}
\usepackage{fancyhdr}
\pagestyle{fancy}
\fancyhf{}
\renewcommand{\sectionmark}[1]{\markright{Section \thesection}}
\renewcommand{\chaptermark}[1]{\markboth{Chapter \thechapter: #1}{}}
\fancyhead[LO,RE]{\normalsize{\rightmark}}
\fancyhead[LE,RO]{\normalsize{\leftmark}}
\fancyfoot[C]{\thepage ${\ }$ v}
\renewcommand{\headrulewidth}{0.5pt}
\renewcommand{\footrulewidth}{0pt}
\setlength{\headsep}{5.0mm}
\setlength{\footskip}{10.0mm}

% new commands:
\newcommand{\elmt}[2]{${^{#2} \textrm{#1}}$}
\newcommand{\elmnt}[3]{${^{#2} \textrm{#1} ^{#3}}$}
\newcommand{\elmf}[2]{^{#2} \textrm{#1}}
\newcommand{\elmnf}[3]{^{#2} \textrm{#1} ^{#3}}
\newcommand{\MeV}[0]{\ \textrm{MeV}}
\newcommand{\AMeV}[0]{\ \textrm{MeV/u}}
\newcommand{\keV}[0]{\ \textrm{keV}}
\newcommand{\AkeV}[0]{\ \textrm{keV/u}}
\newcommand{\GeV}[0]{\ \textrm{GeV}}
\newcommand{\AGeV}[0]{\ \textrm{GeV/u}}
\newcommand{\eV}[0]{\ \textrm{eV}}
\newcommand{\AeV}[0]{\ \textrm{eV/u}}
\newcommand{\mgcm}[0]{\ \textrm{mg} / \textrm{cm} ^{2}}
\newcommand{\Het}[0]{${( ^{3} \textrm{He}, t) \ }$}
\newcommand{\rtb}[0]{${\textrm{R} ^{3} \textrm{B}}$ }
\newcommand{\rtbn}[0]{${\textrm{R} ^{3} \textrm{B}}$}
\newcommand{\rtbp}[0]{${\textrm{R} ^{3} \textrm{B}}$}
\newcommand{\rtbe}[0]{${\textrm{R} ^{3} \textrm{B}}$ experiment }
\newcommand{\rtbep}[0]{${\textrm{R} ^{3} \textrm{B}}$ experiment}
\newcommand{\rtbs}[0]{${\textrm{R} ^{3} \textrm{B}}$ setup }
\newcommand{\rtbsp}[0]{${\textrm{R} ^{3} \textrm{B}}$ setup}
\newcommand{\rtbsNL}[0]{${\textrm{R} ^{3} \textrm{B}}$ meetopstelling }
\newcommand{\rtbsNLp}[0]{${\textrm{R} ^{3} \textrm{B}}$ meetopstelling}
\newcommand{\rtbc}[0]{${\textrm{R} ^{3} \textrm{B}}$ collaboration }
\newcommand{\rtbcp}[0]{${\textrm{R} ^{3} \textrm{B}}$ collaboration }
\newcommand{\lrtb}[0]{${\textrm{LAND} - \textrm{R} ^{3} \textrm{B}}$ }
\newcommand{\lrtbp}[0]{${\textrm{LAND} - \textrm{R} ^{3} \textrm{B}}$}
\newcommand{\lrtbs}[0]{${\textrm{LAND}}$-${\textrm{R} ^{3} \textrm{B}}$ setup }
\newcommand{\lrtbsp}[0]{${\textrm{LAND}}$-${\textrm{R} ^{3} \textrm{B}}$ setup}
\newcommand{\rtbroot}[0]{${\textrm{R} ^{3} \textrm{B}}$Root }
\newcommand{\rtbrootp}[0]{${\textrm{R} ^{3} \textrm{B}}$Root}
\newcommand{\rtbf}[0]{\textrm{R} ^{3} \textrm{B}}
\newcommand{\cel}[0]{\ \degree \textrm{C}}
\newcommand{\parincl}{\mathbin{\!/\mkern-5mu/\!}}
\newcommand{\optcoeff}[2]{ ( #1 | #2 ) }
\newcommand{\whitez}[0]{\textcolor{white}{0}}

% Get right fonts of paragraphs:
\setlength{\parindent}{0cm}
\setlength{\parskip}{10pt}
\setlength{\belowcaptionskip}{5pt}
\setlength{\abovecaptionskip}{5pt}

% Manage floats:
\renewcommand{\floatpagefraction}{.8}
%\renewcommand{\topfraction}{.9}
%\renewcommand{\bottomfraction}{.1}
%\renewcommand{\textfraction}{.01}

% Define our own text colors:
\definecolor{grey}{rgb}{0.4,0.4,0.4}
\definecolor{SkyBlue}{rgb}{0.0,0.9,0.9}
\definecolor{dirtyblue}{rgb}{0.0,0.7,1.0}
\definecolor{orange}{rgb}{1.0,0.6,0.0}
\definecolor{violet}{rgb}{0.6,0.0,1.0}
\definecolor{turquoise}{rgb}{0.0,0.6,0.8}
\definecolor{darkyellow}{rgb}{0.9,0.8,0.0}

% Reading committee adaptations: 
% \definecolor{Olaf}{rgb}{1.0,0.0,0.0}
% \definecolor{Gianluca}{rgb}{0.0,0.0,1.0} 
% \definecolor{Remco}{rgb}{0.0,0.5,0.0}
% \definecolor{Tamii}{rgb}{1.0,0.0,1.0}
% \definecolor{Muhsin}{rgb}{0.42,0.21,0.05}
% \definecolor{Ejiri}{rgb}{1.0,0.5,0.0} 
\definecolor{Olaf}{rgb}{0.0,0.0,0.0}
\definecolor{Gianluca}{rgb}{0.0,0.0,0.0}
\definecolor{Remco}{rgb}{0.0,0.0,0.0}
\definecolor{Tamii}{rgb}{0.0,0.0,0.0}
\definecolor{Muhsin}{rgb}{0.0,0.0,0.0} 
\definecolor{Ejiri}{rgb}{0.0,0.0,0.0} 

% Manage our bibliography:
\usepackage[square,comma,numbers,sort&compress]{natbib}
\usepackage{notoccite}
\bibliographystyle{ieeetr} 
\usepackage[nottoc,notlot,notlof]{tocbibind} % Disables inclusion of table of contents, figures, tables in the contents.

\title{First basic test Document}
\author{Unit Data en Innovatie}
\date{20${^{\textrm{th}}}$ of March 2023}

% Start the document:
\begin{document} 

\chapter{Some piece of text}

Why, my dear, you must know, Mrs. Long says that Netherfield is taken by a young
man of large fortune from the north of England; that he came down on Monday in a
chaise and four to see the place, and was so much delighted with it, that he agreed
with Mr. Morris immediately; that he is to take possession before Michaelmas, and
some of his servants are to be in the house by the end of next week. Not all that
Mrs.




\end{document}
