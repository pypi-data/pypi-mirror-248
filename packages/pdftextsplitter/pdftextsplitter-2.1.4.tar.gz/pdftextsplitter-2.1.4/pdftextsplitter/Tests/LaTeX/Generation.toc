\babel@toc {english}{}
\contentsline {chapter}{\numberline {1}Let's kick-of test-driven development}{2}
\contentsline {section}{\numberline {1.1}Introduction}{2}
\contentsline {section}{\numberline {1.2}Methods}{2}
\contentsline {section}{\numberline {1.3}Methods again}{2}
\contentsline {section}{\numberline {1.4}One more section}{3}
\contentsline {chapter}{\numberline {2}A new chapter starts now}{4}
\contentsline {section}{\numberline {2.1}A new section}{4}
\contentsline {subsection}{\numberline {2.1.1}Nice subsection}{4}
