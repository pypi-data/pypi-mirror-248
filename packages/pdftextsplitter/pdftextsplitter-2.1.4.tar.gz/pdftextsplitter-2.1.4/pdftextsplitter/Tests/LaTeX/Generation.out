\BOOKMARK [1][-]{section.0.1}{Introduction}{}% 1
\BOOKMARK [1][-]{section.0.2}{Methods}{}% 2
\BOOKMARK [0][-]{chapter.1}{A new chapter starts now}{}% 3
\BOOKMARK [1][-]{section.1.1}{A new section}{chapter.1}% 4
\BOOKMARK [2][-]{subsection.1.1.1}{Nice subsection}{section.1.1}% 5
